### ISSUE TITLE
New issues creating/deleting aws_placement_groups

### ISSUE THREAD

**Issue by obourdon** [terraform-providers/terraform-provider-aws#9915]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform: 0.11.14
AWS provider: 2.25.0

### Affected Resource(s)

* aws_placement_group

### Expected Behavior

Until today 2019-08-29 10AM CEST everything was working well when creating/deleting
Placement Group and related/dependant resources. Since today got errors during apply and destroy which tends to prove that AWS API has changed or some states are not taken into account. This reminds me of the recent similar issue #9596

### Actual Behavior

Apply:
```
Error: Error applying plan:

1 error occurred:
	* aws_placement_group.monitor_spread_placement_group: 1 error occurred:
	* aws_placement_group.monitor_spread_placement_group: InvalidPlacementGroup.Unknown: The Placement Groups 'XXX' is unknown.
       status code: 400, request id: ...
```

Destroy:
```
Error: Error applying plan:

1 error occurred:
	* aws_placement_group.monitor_spread_placement_group (destroy): 1 error occurred:
	* aws_placement_group.monitor_spread_placement_group: unexpected state 'available', wanted target 'deleted'. last error: %!s(<nil>)
```

### Steps to Reproduce

Create a placement group and 1 dependant resource. Delete it

### Important Factoids

I am not sure this is 100% reproductible as most "timing/state related issues"

### References

* #9596

**Comment by obourdon** [comment]
Forgot to mention that relaunching apply just after failure just works like a charm

**Comment by obourdon** [comment]
managed to reproduce it with aws provider version 2.38.0

**Comment by obourdon** [comment]
this is becoming more an more frequent. Occured twice over the last 2 days with version 2.43 of AWS provider

**Comment by obourdon** [comment]
Strangely enough, the apply case appeared back today :-(

```Error: Error applying plan:

1 error occurred:
	* aws_placement_group.app_spread_placement_group: 1 error occurred:
	* aws_placement_group.app_spread_placement_group: InvalidPlacementGroup.Unknown: The Placement Group 'XXX' is unknown.
	status code: 400, request id: YYYYYYY
```

**Comment by bitnahian** [comment]
Hi. I'm having a similar issue in ap-southeast-2 region. 

```
resource "aws_placement_group" "web" {
  name     = "test-placement"
  strategy = "cluster"
}
```

On terraform apply, I get the error:

Error: InvalidPlacementGroup.Unknown: The Placement Group 'test-placement' is unknown.
        status code: 400, request id: XXXX

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 2.48.0 of the Terraform AWS Provider, later next week. Thanks to @obourdon for the implementation. 👍
